### ISSUE TITLE
Add whitelisted message

### ISSUE THREAD

**Issue by ejj28** [Minecrell/ServerListPlus#174]
A feature I would like to see would be a customizable whitelist message, that would be shown if you were on/not on the white list. For example:

Welcome to the Minecraft Server!
You are NOT on the whitelist.

or

Welcome to the Minecraft Server!
You are on the whitelist!

Is this possible?
ServerlistMOTD had that feature, but that plugin didn't work right for me so I switched to ServerListPlus. It would be great if you could add a whitelist feature.
Thanks!

**Comment by Minecrell** [comment]
Should be easy to add this at some point, but note that MOTDs for a specific player are not really reliable, since they depend on tracking the player's IP. If the IP of a player changes, they'd get the default MOTD because we wouldn't know if they're whitelisted or not.

**Comment by ejj28** [comment]
Ah, okay! Too bad tracking by UUIDs isn't possible. Thanks!

**Comment by Minecrell** [comment]
Do you no longer need this given what I wrote above? You could have something like:

First time:

```
A Minecraft Server.
Whitelist is active.
```

After joined the first time, one of the following:

```
A Minecraft Server
You are whitelisted!

A Minecraft server
You are NOT whitelisted!
```

It will just not work the first time they look at your server.

**Comment by ejj28** [comment]
Yeah, I don't really need it anymore, thanks though! I have noticed however that if I log in to my server from a different account with the same IP, then on my other account my MOTD says 'Welcome back Y!' when I am actually playing on account X. I'm guessing that this is another downside to the tracking method though, and I don't really care as it's not a big problem. It would be nice though if Mojang would implement proper UUID tracking. Thanks again!
